src/components/BookmarkedNews.tsx

```diff
@@ -0,0 +1,48 @@
+import BookmarkedNewsStateType from '../types/BookmarkedNewsStateType';
+import { connect, ConnectedProps } from 'react-redux'
+import NewsType from '../types/NewsType';
+import News from './News';
+import { removeFromBookmark } from '../redux/actions/bookmarkNewsActions';
+
+type PropsType = {
+  bookmarkedNews: Array<NewsType>
+} & ConnectedProps<typeof connector>
+
+const mapStateToProps = (state: BookmarkedNewsStateType) => {
+  return {
+    bookmarkedNews: state.bookmarkedNews
+  }
+}
+
+const mapDispatchToProps = {
+  removeFromBookmarkedNews: removeFromBookmark
+}
+
+const connector = connect(mapStateToProps, mapDispatchToProps)
+
+function BookmarkedNews({ bookmarkedNews, removeFromBookmarkedNews }: PropsType) {
+  return (
+  <>
+    <div>BookmarkedNews</div>
+    <ul style={{
+          listStyle: 'none'
+        }}>
+        {bookmarkedNews.map( news => 
+          <div key={news.url} >
+            <hr />
```

Comment:
avoid to use <hr/> <br> tags. make use of css instead.

Reply:
Resolved